tests/baselines/reference/functionsMissingReturnStatementsAndExpressions.errors.txt

```diff
@@ -112,6 +112,14 @@ tests/cases/compiler/functionsMissingReturnStatementsAndExpressions.ts(127,5): e
         // Not okay; union does not contain void or any
     }
     
+    function f22(): undefined {
+        // Okay; return type allows implicit return of undefined
+    }
+    
+    function f23(): undefined | number {

```

Comment:
I think your last commit only included this baseline, as f23 doesn't appear to be anywhere else.

Reply:
I just tapped the commit button from GitHub, will add it with the another change for the feedback.